serve/source_v1_test.go

```diff
@@ -104,77 +106,85 @@ func TestSourceSuccess(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Failed to dial bufnet: %v", err)
 	}
-	c, err := clients.NewClient(ctx, specs.RegistryGrpc, "", "", clients.WithGRPCConnection(conn), clients.WithNoSentry())
```

Comment:
How does the `--no-sentry` option get passed to the plugins now? 🤔

Reply:
I guess the code for managing the plugins will now live in the CLI?

Reply:
Yes.

Reply:
For the test because it's already running sentry is not relevant here.